static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -118,6 +147,55 @@ class RuleConditionsForm extends PureComponent<Props, State> {
     this.fetchData();
   }
 
+  getFilterKeys = () => {
+    const {organization, dataset, tags} = this.props;
+    const {measurements} = this.state;
+    const measurementsWithKind = Object.keys(measurements).reduce(
+      (measurement_tags, key) => {
+        measurement_tags[key] = {
+          ...measurements[key],
+          kind: FieldKind.MEASUREMENT,
+        };
+        return measurement_tags;
+      },
+      {}
+    );
+    const orgHasPerformanceView = organization.features.includes('performance-view');
+    const combinedTags: TagCollection =
+      dataset === Dataset.ERRORS
+        ? Object.assign({}, [], STATIC_FIELD_TAGS_WITHOUT_TRANSACTION_FIELDS)
```

Comment:
i think i'd use spread instead, also i'm not sure the array does anything here
```suggestion
        ? Object.assign({}, STATIC_FIELD_TAGS_WITHOUT_TRANSACTION_FIELDS)
```